cognite/client/_api/transformations.py

```diff
@@ -0,0 +1,324 @@
+from typing import Awaitable, List, Optional, Union
+
+from cognite.client import utils
+from cognite.client._api.transformation_jobs import TransformationJobsAPI
+from cognite.client._api.transformation_notifications import TransformationNotificationsAPI
+from cognite.client._api.transformation_schedules import TransformationSchedulesAPI
+from cognite.client._api.transformation_schema import TransformationSchemaAPI
+from cognite.client._api_client import APIClient
+from cognite.client.data_classes import Transformation, TransformationJob, TransformationList
+from cognite.client.data_classes.transformations import (
+    TransformationFilter,
+    TransformationPreviewResult,
+    TransformationUpdate,
+)
+
+
+class TransformationsAPI(APIClient):
+    _RESOURCE_PATH = "/transformations"
+    _LIST_CLASS = TransformationList
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.jobs = TransformationJobsAPI(*args, **kwargs)
+        self.schedules = TransformationSchedulesAPI(*args, **kwargs)
+        self.schema = TransformationSchemaAPI(*args, **kwargs)
+        self.notifications = TransformationNotificationsAPI(*args, **kwargs)
+
+    def create(
+        self, transformation: Union[Transformation, List[Transformation]]
+    ) -> Union[Transformation, TransformationList]:
+        """`Create one or more transformations. <https://docs.cognite.com/api/playground/#operation/createTransformations>`_
+
+        Args:
+            transformation (Union[Transformation, List[Transformation]]): Transformation or list of transformations to create.
+
+        Returns:
+            Created transformation(s)
+
+        Examples:
+
+            Create new transformations:
+
+                >>> from cognite.experimental import CogniteClient
+                >>> from cognite.experimental.data_classes import Transformation, TransformationDestination
+                >>> c = CogniteClient()
+                >>> transformations = [
+                >>>     Transformation(
+                >>>         name="transformation1",
+                >>>         destination=TransformationDestination.assets()
+                >>>     ),
+                >>>     Transformation(
+                >>>         name="transformation2",
+                >>>         destination=TransformationDestination.raw("myDatabase", "myTable"),
+                >>>     ),
```

Comment:
can we make this more detailed, one with all required fields? we can do this after the documentation is added to `rst` file.